src/posts/events/hackawayv5/index.mdx

```diff
@@ -109,8 +109,8 @@ import { hackaway5logoClass } from "./v5.module.scss"
     <EventFullDescription />
   </p>
   <ButtonsContainer className="justify-content-center">
-    <SiteButton href="https://forms.office.com/Pages/ResponsePage.aspx?id=0AAuJQ_m3UeQ9lpqzysIsEfppBoOPM5HiYSEN-63P_lUM0hJUDUzRVNTRzZYV0kyUzNJWVpVRUZaRC4u" className="btn-hackaway-orange">
-      Register your interest
+    <SiteButton href="/tickets" className="btn-hackaway-orange">
```

Comment:
When using the SiteButton component, use the `to=` parameter instead of `href=` so there is no page refresh